src/pages/index.js

```diff
@@ -80,64 +74,185 @@ const addCardPopup = new PopupWithForm(
 );
 addCardPopup.setEventListeners();
 
+
+//  
+// Image Preview Popup
+//  
+
 const popupImageModal = new PopupWithImage('#preview-image-modal');
 popupImageModal.setEventListeners();
 
-const cardSection = new Section(
-	{
-		items: initialCards,
-		renderer: renderCard,
-	},
-	'.cards__list'
-);
-cardSection.renderItems();
-
-// function openPopup(modal) {
-// 	modal.classList.add('modal_opened');
-// 	document.addEventListener('keydown', escPopup);
-// }
-
-// function closePopup(modal) {
-// 	modal.classList.remove('modal_opened');
-// 	document.removeEventListener('keydown', escPopup);
-// }
-
-// function escPopup(e) {
-// 	if (e.key === 'Escape') {
-// 		const modal = document.querySelector('.modal_opened');
-// 		if (modal) closePopup(modal);
-// 	}
-// }
-
-function renderCard(cardData) {
-	const card = new Card(cardData, '#card-template', handleImageClick);
-	const cardElement = card.getView();
-	cardSection.addItem(cardElement);
+function handleImageClick(name, link) {
+	popupImageModal.open({ name, link });
 }
 
-function handleOverlayClose(e) {
-	if (e.target.classList.contains('modal_opened')) {
-		popupImageModal.close();
-	}
+//  
+// Form Validation
+//  
+
+const addEditValidator = new FormValidator(config, editFormElement);
+addEditValidator.enableValidation();
+
+const addCardValidator = new FormValidator(config, addFormElement);
+addCardValidator.enableValidation();
+
+const updateAvatarValidator = new FormValidator(config, avatarFormElement);
+updateAvatarValidator.enableValidation();
+
+//  
+// User Info
+//  
+
+const userInfo = new UserInfo({
+	nameSelector: '.profile__title',
+	jobSelector: '.profile__description',
+	userAvatar: '.profile__image',
+});
+
+// 
+// Functions
+//  
+
+function createCard(cardData) {
+	const card = new Card(
+		cardData,
+		'#card-template',
+		handleImageClick,
+		handleTrashClick,
+		handleLikeClick,
+	);
+	cardSection.addItem(card.generateCard());
```

Comment:
It's highly recommended to make a separate function `getCard` for creating a card not to copy-paste the code if you want to add the card with **append** in future

```js		

function getCard(item) {
    // here you create a card
  return cardElement.generateCard()
}

```


So, here will be only 2 lines of code


```js

function createCard(cardData) {
  const card = getCard(cardData);
  cardSection.addItem(card);
}
```